app/assets/stylesheets/styles/guides/_new_guides.css.scss

```diff
@@ -1,5 +1,4 @@
-// You can't nest keyframes
-
+// TODO: This should not be here. This makes me unhappy.
 @-webkit-keyframes fa-spin-image-load {
```

Comment:
I couldn't figure out what to do with this. Maybe we just need our own keyframe and not use `fa-spin` in this case?